service/worker/workerdeployment/client.go

```diff
@@ -726,11 +752,11 @@ func (d *ClientImpl) updateWithStartWorkerDeploymentVersion(
 			},
 			CreateTime:        now,
 			RoutingUpdateTime: now,
```

Comment:
I don't think `RoutingUpdateTime` this should have any value at the beginning because no setCurrent or getCurrent was called for this version yet.